examples/joke_bot/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "misterbot"
+version = "0.1.0"
+authors = ["Isaiah Inuwa <isaiah.inuwa@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+ruma = { version = "0.4.0", path = "../../crates/ruma", features = ["client-api-c", "client", "client-hyper-native-tls", "events"] }
+ruma-client = { version = "0.7.0", path = "../../crates/ruma-client", features = ["client-api"]}
+# For building locally: use the git dependencies below.
+# Browse the source at this revision here: https://github.com/ruma/ruma/tree/f161c8117c706fc52089999e1f406cf34276ec9d
+# ruma = { git = "https://github.com/ruma/ruma", rev = "f161c8117c706fc52089999e1f406cf34276ec9d", features = ["client-api-c", "client", "client-hyper-native-tls", "events"] }
+# ruma-client = { git = "https://github.com/ruma/ruma", rev = "f161c8117c706fc52089999e1f406cf34276ec9d", features = ["client-api"] }
```

Comment:
Personally i would recommend removing these, actually

Reply:
> Are these git rev's still up-to-date?

Yeah, I bumped it just before squashing.

> Personally i would recommend removing these, actually

@ShadowJonathan, I think the way we do versions makes copy/pasting the example confusing to users. Since we don't bump versions until a new release, the example needs to be kept up to date with relevant breaking changes in `main` even though `Cargo.toml` refers to the previous release.

Because of that, it can be helpful for someone trying to adapt their own bot based on this code to know that the example is based on an unreleased version of Ruma. Adding the git rev as a comment makes it easy for them to check out the state of the repo. The downside is that the git rev may not be kept up to date...